### ISSUE TITLE
[BUG/ISSUE] Wrong path from download_data.py script

### ISSUE THREAD

**Issue by IshirDutta** [geoschem/geos-chem#1347]
### What institution are you from?
MIT

### Description of the problem
Trying to download cropped MERRA-2 files using dryrun appends './' before the write directory, causing the files to be written to 

`‘.//net/geoschem/data/gcgrid/data/ExtData/GEOS_0.5x0.625_NA/MERRA2/2017/’`
instead of 
`‘/net/geoschem/data/gcgrid/data/ExtData/GEOS_0.5x0.625_NA/MERRA2/2017/'`

### Description of troubleshooting performed
I tried to duplicate the issue in v13.3.4 (with which I've previously been able to download files using the dryrun script), which worked correctly, but I'm not sure that it's a perfect 1:1 comparison as I believe it's the last version in which HEMCO_Config.rc needed line-by-line edits to the MERRA-2 read-ins. 

```
./gcclassic --dryrun > log.dryrun
./download_data.py log.dryrun --wu
```

I also compared the two log.dryrun.unique files to see if there were differences, but they seem to be pulling the correct paths out of log.dryrun. 

I then uncommented the section of download_data.py that deletes the shell script upon completion and saw that the wget command list does in fact have the offending './'
A diff of download_data.yml revealed minor changes to the ComputeCanada "command" line, but it was just extra spaces being deleted.

### GEOS-Chem version
13.4.1

### Description of modifications
Describe your modifications here.

### Log files
Python data download: 
[download_data.py.txt](https://github.com/geoschem/geos-chem/files/9311155/download_data.py.txt)
[download_data_1334.py.txt](https://github.com/geoschem/geos-chem/files/9311156/download_data_1334.py.txt)

YAML files
[download_data.yml.txt](https://github.com/geoschem/geos-chem/files/9311160/download_data.yml.txt)
[download_data_1334.yml.txt](https://github.com/geoschem/geos-chem/files/9311161/download_data_1334.yml.txt)

Dryrun unique
[log.dryrun.unique.txt](https://github.com/geoschem/geos-chem/files/9311167/log.dryrun.unique.txt)
[log.dryrun.unique_1334.txt](https://github.com/geoschem/geos-chem/files/9311168/log.dryrun.unique_1334.txt)

v13.4.1 Shell script
[auto_generated_download_script.sh.txt](https://github.com/geoschem/geos-chem/files/9311172/auto_generated_download_script.sh.txt)

### Software versions
- CMake version:
- Compilers (Intel or GNU, and version):
- NetCDF version:

**Comment by yantosca** [comment]
Thanks @IshirDutta.  We will look into this and can patch this in 14.0.1.

**Comment by Jourdan-He** [comment]
I have not yet been able to replicate this due to container issue, so I built a new container last week, will work on it this week.

**Comment by msulprizio** [comment]
Thanks for the update @Jourdan-He. I will remove the 14.0.1 milestone from this so we can proceed with releasing that version. A fix can go into either 14.0.2 or 14.1.0.
